### DISCUSSION TITLE
Pas de cote du tournoi des 6 nations

### DISCUSSION THREAD

**Discussion by kreon76** [pretrehr/Sports-betting#19] [General]
Rebonjour,

Suis-je le seul impacté ? quand je fais un scrap du tournoi des 6 nations de Rugby je n'ai aucune cote

![image](https://user-images.githubusercontent.com/78261069/107799614-ef5a7100-6d5d-11eb-821a-7156fa401855.png)

J'en ai fait 3 de suite toujours pareil.

Merci par avance

Cdt

**Comment by pretrehr** [comment]
Je n'ai pas ce problème de mon côté...
Vérifiez que vous avez bien sélectionné "Six Nations" et pas "Six Nations (F)".
Y a-t-il un message d'erreur dans le terminal ?
Sinon vous pouvez toujours sélectionner uniquement l'option "Tout le rugby" dans la liste des compétitions.

**Reply by kreon76** [reply]
Je sélectionne bien Six Nations mais ce n'est pas grave car effectivement si je prends tout le rugby ça fonctionne, merci
